tools/vms-generator/utils/utils.go

```diff
@@ -98,6 +98,23 @@ var DockerTag = "devel"
 
 var gracePeriod = int64(0)
 
+var VMCidrs = map[string]struct {
```

Comment:
s/VMCidrs/protocolToVmIps

Reply:
I removed this commit, I will continue with this in a new PR